commcare_connect/organization/views.py

```diff
@@ -58,3 +70,19 @@ def accept_invite(request, org_slug, invite_id):
     membership.save()
     messages.success(request, message=f"Accepted invite for joining {organization.slug} organization.")
     return redirect("organization:home", org_slug)
+
+
+@org_admin_required
+@require_POST
+def add_credential_view(request, org_slug):
+    org = get_object_or_404(Organization, slug=org_slug)
+    credentials = connect_id_client.fetch_credentials()
+    if not any(c.name == f"Worked for {org.name}" for c in credentials):
+        credentials.append(Credential(name=f"Worked for {org.name}", slug=f"Worked for {org.name}"))
```

Comment:
why does the slug differ between the two places this occurs?

Reply:
Additionally, slugs should be an actual slug not normal text (lowercased, underscore separated, etc).

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/323/commits/9aa3e50ceef5c1575cb6bf9733cdf91a96cef236